app/controllers/posts_controller.rb

```diff
@@ -1,8 +1,10 @@
 class PostsController < ApplicationController
+  include HackernewsClone::VoteHelper
```

Comment:
this appears to only be used in this controller. plans to expand in the future?

Reply:
yes - it'll be used by Comments